packages/overlay/src/vaadin-overlay-position-mixin.js

```diff
@@ -232,9 +232,11 @@ export const PositionMixin = (superClass) =>
 
       const targetRect = this.positionTarget.getBoundingClientRect();
 
-      if (targetRect.width === 0 && targetRect.height === 0 && this.opened) {
-        this.opened = false;
+      if (targetRect.width === 0 && targetRect.height === 0) {
+        this.style.display = 'none';
```

Comment:
Could use `this.hidden = targetRect.width === 0 && targetRect.height === 0;` instead of an inline style

Reply:
This is a cleaner way. Changed.